modules/ROOT/pages/basic-alerts.adoc

```diff
@@ -40,4 +40,79 @@ The following levels of severity are available for basic alerts:
 
 include::partial$include-alert-severity.adoc[leveloffset=+1]
 
-// add basic alert email templates
+[[alert-variables]]
+== Basic Alert Email Template
+
+You can use the following variables in a basic alert email's *Subject* and *Message* fields.
+For the description of each variable, see <<var-defs>>.
+
+[%header,cols="25a,50a,20a"]
+.Variables for Alerts
+|===
+| Alert Type | Alert | Applicable Variables
+2+| All Alerts | `${resource}` `${severity}` `${state}`
+.7+| CloudHub App Alerts |*CPU usage - CloudHub* .2+|`${operator}` `${value}`
+| *Memory usage - CloudHub*
+| *Custom application notification* | `${priority}` `${message}`
+| *Deployment failed* |  `${user}` `${message}`
+| *Deployment success* .2+| `${user}`
+| *Worker not responding*
+| *Exceeds event traffic threshold* | `${period}`
+
+
+.4+|Hybrid App Alerts | All Hybrid App Alerts| `${name}`
+| *Number of Mule messages* .3+| `${value}` `${threshold}`
+| *Number of errors*
+| *Response time*
+
+.5+|Server Alerts | All Server Alerts | `${name}`
+| *Server CPU usage* .4+|  `${value}` `${comparison}` `${threshold}`
+| *Server Load average*
+| *Server Memory average*
+| *Server Thread count*
+
+.9+| Server Group Alerts| All Server Group Alerts | `${name}`
+| *Server Group's node CPU usage* .4+|  `${targetName}` `${value}` `${comparison}` `${threshold}`
+| *Server Group's node Load average*
+| *Server Group's node Memory usage*
+| *Server Group's node Thread count*
+| *Server Group's node connected* .4+| `${targetName}`
+| *Server Group's node disconnected*
+| *Server added to Server Group*
+| *Server removed from Server Group*
+
+.9+| Cluster Alerts |All Cluster Alerts| `${name}`
+| *Cluster's node CPU usage* .4+| `${targetName}` `${value}` `${comparison}` `${threshold}`
+| *Cluster's node Load average*
+| *Cluster's node Memory usage*
+| *Cluster's node Thread count*
+| *Cluster's node disconnected* | `${resource}` `${targetName}` 
+| *Cluster's node connected* .3+| `${targetName}` 
+| *Server added to Cluster*
+| *Server removed from Cluster*
+|===
+
+
+[%header%autowidth.spread]
+[[var-defs]]
+.Variable Definitions
+|===
+|Variable |Definition
+|`${alertLink}`|URL of the triggered alert
+|`${alertState}`|State of the alert when it was triggered. Available values are `OK` or `Alerting`.
+|`${comparison}`|Above or Below
+|`${condition}`|Metric for which the alert is triggered
+|`${environment}`|Name of the environment in which the resource is deployed
+|`${operator}` |An operator, such as `< > =`
+|`${organization}` |Name of the organization that owns the alert
+|`${period}` |Period of time over which to measure
+|`${resource}` a| Name of the item that triggers the alert:
+
+* CloudHub Apps: Name of the app that triggers the alert
+* Hybrid Apps: Name of the resource in the app that triggers the alert
+|`${severity}` | Severity of the alert
+|`${targetName}`|Name of the node in a server group or cluster
```

Comment:
targetName: Name of the server, server group, or cluster
memberName: Name of the server within the server group or cluster